Sources/MapboxCoreNavigation/DispatchTimer.swift

```diff
@@ -5,12 +5,28 @@ import Dispatch
  `DispatchTimer` is a general-purpose wrapper over the `DispatchSourceTimer` mechanism in GCD.
  */
 public class DispatchTimer {
+    /**
+     Describes related `DispatchTimer`'s state.
+     */
```

Comment:
Thanks for noticing that this was undocumented.

```suggestion
    /**
     The state of a `DispatchTimer`.
     */
```